hvplot/converter.py

```diff
@@ -446,20 +446,19 @@ def __init__(
                         "Projection must be defined as cartopy CRS or "
                         f"one of the following CRS string:\n {all_crs}")
 
-            projection = projection or (ccrs.GOOGLE_MERCATOR if tiles else self.crs)
-            if tiles and projection != ccrs.GOOGLE_MERCATOR:
+            self.proj_crs = projection or (ccrs.GOOGLE_MERCATOR if tiles else self.crs)
+            if tiles and self.proj_crs != ccrs.GOOGLE_MERCATOR:
```

Comment:
To help future developers understand, should we rename `proj_crs` to `feature_projection`?